post_installation_configuration/configuring-multi-arch-compute-machines/multiarch-tuning-operator.adoc

```diff
@@ -12,21 +12,29 @@ The Multiarch Tuning Operator is a Technology Preview feature only. Technology P
+// This Operator implements the `ClusterPodPlacementConfig` object to support architecture-aware workload scheduling.
+// For more information about manually scheduling worloads on multi-architecture clusters, see "Sample multi-architecture node workload deployments".
 
-When you create the `ClusterPodPlacementConfig` object, the Multiarch Tuning Operator deploys an operand. This operand automates the configuration of scheduling predicates for the workloads based on their supported architectures. It uses the `kubernetes.io/arch` label set by the kubelet in the node objects.
+You can use the Multiarch Tuning Operator to automatically schedule workloads on nodes based on their supported architectures, instead of manually scheduling them.
 
-When a pod is created, the operand configures the `nodeAffinity` field in the pod specification by computing the subset of architectures that the pod can support.
+To enable architecture-aware workload scheduling, you must create the `ClusterPodPlacementConfig` object by using the Multiarch Tuning Operator. When you create the `ClusterPodPlacementConfig` object, this Operator deploys an operand.
 
-Before a pod enters the scheduling cycle, the operand adds the `multiarch.openshift.io/scheduling-gate` scheduling gate that prevents the scheduling of the pod until the operand completes the following actions:
+When a pod is created, the operand deployed by this Operator schedules workloads on nodes based on their supported architectures by performing the following actions:  
 
-. Compute a predicate for the `kubernetes.io/arch` label. 
+. Add the `multiarch.openshift.io/scheduling-gate` scheduling gate to prevent the scheduling of the pod.
+. Compute a predicate based on the `kubernetes.io/arch` label in the node objects. 
 . Integrate the predicate as a `nodeAffinity` requirement in the pod specification.
-. Remove the scheduling gate.
+. Remove the scheduling gate from the pod.
 
-These operand actions restrict the scheduling of workloads to those nodes that have the supported architectures.
+When the operand removes the scheduling gate, the pod enters the scheduling cycle. The workload is then scheduled on nodes based on the supported architectures specified in the `nodeAffinity` field of the pod specification.
+
+// This operand automates the configuration of scheduling predicates for the workloads based on their supported architectures. It uses the `kubernetes.io/arch` label set by the kubelet in the node objects. When a pod is created, the operand configures the `nodeAffinity` field in the pod specification by computing the subset of architectures that the pod can support. When a pod is created, this operand adds the `multiarch.openshift.io/scheduling-gate` scheduling gate that prevents the scheduling of the pod until the operand completes the following actions:
+
+// These operand actions restrict the scheduling of workloads to those nodes that have the supported architectures.
```

Comment:
This is commented and not visible, right?

Reply:
Yes.  I will delete these comments when the updates are finalized.